### ISSUE TITLE
Minimum OS Requirement Message on Big Sur

### ISSUE THREAD

**Issue by JeffreyBytes** [JayBrown/DiMaGo#1]
When using DiMaGo > Create on Big Sur I get the following message:
Error! Minimum OS Requirement: OS X 10.8 (Mountain Lion)
![Screen Shot 2020-08-14 at 02 20 35](https://user-images.githubusercontent.com/990135/90224251-003d2080-ddd5-11ea-9d09-ff37d5da2ba7.png)

I know Big Sur is still in beta, just thought you should know.

**Comment by JayBrown** [comment]
Yeah, it's BS checking the minor version number, and seeing "0", which is less than "8". However, it's a simple fix. You just need to put `export SYSTEM_VERSION_COMPAT=1` at the beginning of the script, after the shebang, and then the OS version number will print "10.16" instead of "11.0".

**Comment by piratx** [comment]
Still works on Monterey! :)
